Update Neovim installation instructions (#68)
ionide/tree-sitter-fsharp: README.md

The grammar starts with the `file` node at the beginning of the rules.

### Adding to neovim

tree-sitter-fsharp isn't supported out-of-the box by [nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter).
However, nvim-treesitter has [instructions for how to manually add unsupported parsers](https://github.com/nvim-treesitter/nvim-treesitter?tab=readme-ov-file#adding-parsers).

For tree-sitter-fsharp, this involves three steps:

1. Update your Neovim config for nvim-treesitter to refer to tree-sitter-fsharp.
2. Run `:TSInstall fsharp` inside Neovim.

> 3. Copy the files from [./queries/](./queries) to the ./queries/fsharp/ directory of your Neovim runtime directory (where your config is stored) - see [the Adding queries section of the nvim-treesitter README](https://github.com/nvim-treesitter/nvim-treesitter?tab=readme-ov-file#adding-queries).

Reviewer: I would suggest to just use the full path to avoid any ambiguity :) 

```suggestion
3. Copy the files from [./queries/](./queries) to the neovim config directory at`$XDG_CONFIG_HOME/nvim/queries/fsharp/` - see [the Adding queries section of the nvim-treesitter README](https://github.com/nvim-treesitter/nvim-treesitter?tab=readme-ov-file#adding-queries).
```
